java/arcs/core/host/ExternalHost.kt

```diff
@@ -1,9 +1,18 @@
 package arcs.core.host
 
+import arcs.core.sdk.Particle
+import kotlinx.coroutines.runBlocking
+
 /**
- * [ExternalHost] is the base class for all Android-specific non-isolated hosts.
+ * [ExternalHost] is the base class for all Platform-specific non-isolated hosts.
  * Just a place holder for now, but serves as a marker for the [Allocator]
- * in the future, and likely to house Android specific state, like application
- * context.
+ * in the future, and likely to house Android or Browser specific state, like application
+ * context or browser global scope.
  */
-class ExternalHost : AbstractArcHost()
+open class ExternalHost(vararg externalParticles: Particle) : AbstractArcHost() {
+    init {
+        runBlocking {
+            externalParticles.forEach { registerParticle(ParticleIdentifier.from(it::class)) }
+        }
+    }
```

Comment:
If this is the best way to perform initialization (with a blocking call to suspending functions), please make it abundantly clear in the KDoc that `ExternalHost`s should never be instantiated on the main thread.

Reply:
I made register/unregister a private API, since most Android/External hosts will have their own builtin list and won't allow external removal/add. Only ProdHost needs this, as DevHost delegates to TS to discover them as well. Therefore I removed the coroutines for this.